module_cheatsheets/scRNA-seq_cheatsheet.md

```diff
@@ -0,0 +1,81 @@
+| `SingleCellExperiment`               | [`logcounts()`](https://bioconductor.org/packages/devel/bioc/vignettes/scater/inst/doc/vignette-intro.html)           | Logcounts      | Stores or extracts log transformed single cell experiment count data as an assay of the SingleCellExperiment object       |
+| `SingleCellExperiment` | [`SingleCellExperiment()`](https://bioconductor.org/packages/devel/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html#2_creating_singlecellexperiment_instances)                | Single Cell Experiment                     | Creates a `SingleCellExperiment` object                         |
+| `SingleCellExperiment`        | [`rowData()`](https://bioconductor.org/packages/devel/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html)          | Row Data   | Extracts and stores gene-level metadata that describes features of the `SingleCellExperiment` object        |
+
+
+<div style="page-break-after: always;"></div>
+
+### `alevinQC`, `colorblindr`, `Rtsne`, `tibble`
+Documentation for each of these packages can be accessed by clicking the package name in the table below.
+
+| Library/Package      | Piece of Code                           | What it's called                                                              | What it does                                                                                                           |
+|----------------------|-----------------------------------------|-------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------|
+| [`alevinQC`](http://www.bioconductor.org/packages/devel/bioc//vignettes/alevinQC/inst/doc/alevinqc.html)             | [`alevinQCReport()`](http://www.bioconductor.org/packages/devel/bioc//vignettes/alevinQC/inst/doc/alevinqc.html#generate-qc-report)                      | Alevin QC Report                                                              | Produces a QC (quality check) report from the alevin output                                                           |
+| [`colorblindr`](https://www.rdocumentation.org/packages/colorblindr/versions/0.1.0)  |        [`scale_color_OkabeIto()`](https://www.rdocumentation.org/packages/colorblindr/versions/0.1.0/topics/scale_colour_OkabeIto)                      | OkabeIto Color Scale    | When added as a layer to a plot, makes the plot color-blind friendly                                                    |
+| [`Rtsne`](https://www.rdocumentation.org/packages/Rtsne/versions/0.15)                | [`Rtsne()`](https://www.rdocumentation.org/packages/Rtsne/versions/0.15/topics/Rtsne)                               | T-Distributed Stochastic Neighbor Embedding using a Barnes-Hut Implementation | Reduces the dimensions of the specified matrix or data frame                                                        |
```

Comment:
Do we want to add something here and potentially above for `prcomp` about how to extract the scores? Or perhaps we just leave that to the notebook itself?

Reply:
It may be helpful to add an explanation on how to extract the scores for both of those functions, but I do not think that the explanation would need its own row. What do you think about adding something brief to the end of the `Rtsne` and `prcomp` definitions for this purpose?

Reply:
Eh. Actually maybe that's where the participants should either utilize the notebooks and/or use the documentation link. Thoughts?

Reply:
True, the cheatsheets are meant to present brief descriptions of each function, not necessarily explanations on how to use them. Let's leave it to the notebooks and documentation, the method to extract scores is pretty clear in the notebook.